tests/users/test_api_update_user.py

```diff
@@ -174,6 +188,83 @@ def test_update_availability_to_be_mentee_to_false(self):
         self.assertDictEqual(expected_response, json.loads(actual_response.data))
         self.assertEqual(test_need_mentoring, self.first_user.need_mentoring)
 
+    def test_change_password_to_different_password(self):
+        self.add_user()
+        auth_header = get_test_request_header(self.first_user.id)
+        expected_response = messages.PASSWORD_SUCCESSFULLY_UPDATED
+        actual_response = self.client.put(
+            '/user/change_password', follow_redirects=True,
+            headers=auth_header,
+            data=json.dumps(dict(current_password=user1['password'],
+                                 new_password=user2['password'])),
+            content_type='application/json')
+
+        self.assertEqual(201, actual_response.status_code)
+        self.assertDictEqual(expected_response,
+                             json.loads(actual_response.data))
+
+    def test_change_password_to_empty_password(self):
+        self.add_user()
+        auth_header = get_test_request_header(self.first_user.id)
+        expected_response = messages.WRONG_LENGTH_OF_PASSWORD
+        actual_response = self.client.put(
+            '/user/change_password', follow_redirects=True,
+            headers=auth_header,
+            data=json.dumps(dict(current_password=user1['password'],
+                                 new_password="")),
+            content_type='application/json')
+
+        self.assertEqual(400, actual_response.status_code)
+        self.assertDictEqual(expected_response,
+                             json.loads(actual_response.data))
+
+    def test_change_password_to_password_with_white_spaces(self):
+        self.add_user()
+        auth_header = get_test_request_header(self.first_user.id)
+        expected_response = messages.USER_INPUTS_SPACE_IN_PASSWORD
+        actual_response = self.client.put(
+            '/user/change_password', follow_redirects=True,
+            headers=auth_header,
+            data=json.dumps(dict(current_password=user1['password'],
+                                 new_password="n e w  p a s s w o r d")),
+            content_type='application/json')
+
+        self.assertEqual(400, actual_response.status_code)
+        self.assertDictEqual(expected_response,
+                             json.loads(actual_response.data))
+
+    def test_change_password_without_token(self):
+        self.add_user()
+        auth_header = {'Authorization': 'Bearer {}'.format("")}
+        expected_response = messages.TOKEN_IS_INVALID
```

Comment:
Here auth token should be missing and message should be `AUTHORISATION_TOKEN_IS_MISSING`